app/graph/graphql_crud_operations.rb

```diff
@@ -315,19 +309,19 @@ def self.define_default_type(&block)
     GraphQL::ObjectType.define do
       global_id_field :id
 
-      field :permissions, types.String do
+      field :permissions, types.String, 'CRUD permissions of this record for current user' do
         resolve -> (obj, _args, ctx) {
           obj.permissions(ctx[:ability])
         }
       end
 
-      field :created_at, types.String do
+      field :created_at, types.String, 'Datetime of record creation' do
         resolve -> (obj, _args, _ctx) {
           obj.created_at.to_i.to_s if obj.respond_to?(:created_at)
         }
       end
 
-      field :updated_at, types.String do
+      field :updated_at, types.String, 'Datetime of record last update' do
         resolve -> (obj, _args, _ctx) {
           obj.updated_at.to_i.to_s if obj.respond_to?(:updated_at)
         }
```

Comment:
Not sure if you're planning to make this change in the future, but those two fields should be converted to integer

Reply:
I would also make clear that the datetime format is a timestamp in seconds